upstream.wrap

```diff
@@ -0,0 +1,7 @@
+[wrap-file]
+source_url = https://github.com/FluidSynth/fluidsynth/archive/v2.1.7.tar.gz
```

Comment:
```
$ mesonwrap review fluidsynth --pull-request 2 --allow-other-files
[...]
upstream.wrap has directory: ✗
```
https://mesonbuild.com/Wrap-dependency-system-manual.html#accepted-configuration-properties-for-wraps

-    `directory` - name of the subproject root directory, defaults to the name of the wrap.

Needs to be fluidsynth-2.1.7